packaging/Dockerfile.centos.7

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2018-2020, Intel Corporation
```

Comment:
Is there any reason why you use 2020 instead of 2021? Also, why do you use different years in the files added in the packaging directory (2018-2019, 2018-2020, 2019, etc.)?

Reply:
The code is inherit from packaging, cant really tell you what is this about.